src/meta/metainfo.go

```diff
@@ -299,6 +299,62 @@ type FuncParam struct {
 type PhpDocInfo struct {
 	Deprecated      bool
 	DeprecationNote string
+
+	Removed     bool
+	RemovalNote string
+}
+
+func (p PhpDocInfo) StringForFunction(funcName string) string {
+	if funcName == "" {
+		return ""
+	}
+	return p.string(funcName, NewEmptyTypesMap(0), "")
+}
+
+func (p PhpDocInfo) StringForMethod(exprType TypesMap, methodName string) string {
+	if methodName == "" {
+		return ""
+	}
+	return p.string("", exprType, methodName)
+}
+
+func (p PhpDocInfo) string(funcName string, exprType TypesMap, methodName string) string {
```

Comment:
I'm not that comfortable with a method name that is the same as builtin type name.
But please don't call it `String()` as that method usually more general than giving a specific printing format for reports.
You may format the message somewhere in the linter code.